### PR TITLE
Support React 15.0.x

### PR DESCRIPTION
**Pull request by ayrton**
Adds support for the latest version of React (while still supporting 0.14.x)

### CODE REVIEW

File: src/ReactTestWrapper.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   value () {
-    return this.attr('value')
+    return this.el.value
```
Comment (ayrton) [inline src/ReactTestWrapper.js]:
```jsx

<input defaultValue="hello world" />

this.attr('value') // => undefined
this.el.value // => 'hello world'
```

Tested on older version (0.14.x) and everything still works as expected.

**Comment by vesln** [comment]
:+1:

File: package.json [hunk 2]
```diff
@@ -57,9 +57,9 @@
     "enzyme": "^2.0.0",
     "jsdom": "^8.0.4",
     "mocha": "^2.4.5",
-    "react": "^0.14.7",
-    "react-addons-test-utils": "^0.14.7",
-    "react-dom": "^0.14.7",
+    "react": "^0.14.0 || ^15.0.0-0",
```
Comment (ljharb) [inline package.json]: Why is the -0 needed?

Comment (ayrton) [inline package.json]: Also includes release candidates

Comment (ljharb) [inline package.json]: And this semver range will include 15.0.1 etc?

Comment (ayrton) [inline package.json]: Yep it sure does 😄:

```js
> semver = require('semver')
// ...
> semver.satisfies('15.0.1', '^0.14.0 || ^15.0.0-0')
true
> semver.satisfies('15.0.0-rc1', '^0.14.0 || ^15.0.0-0')
true
> semver.satisfies('0.14.0', '^0.14.0 || ^15.0.0-0')
true
```

**Comment by keithamus** [comment]
:+1:

**Pull request merged**